yaml/_type/timestamp.ts

```diff
@@ -68,7 +68,7 @@ function constructYamlTimestamp(data: string): Date {
 
   let delta = null;
   if (match[9]) {
-    const tzHour = +match[10];
+    const tzHour = +match[10]!;
```

Comment:
Can you please justify this non-null assertion? The other assertions look fine.

Reply:
I thought that the check of `match[9]` was enough to establish that a time zone is present, but maybe an explicit check on `match[10]` is better. I will change this.